AtlasStarter/Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using MongoDB.Driver;
+
+namespace AtlasStarter
+{
+    class MainClassNoPrompt
+    {
+        public static void Main(string[] args)
+        {
+            // TODO:
+            // Replace the placeholder connection string below with your
+            // Altas cluster specifics. Be sure it includes
+            // a valid username and password! Note that in a production environment,
+            // you do not want to store your password in plain-text here.
+
+            var mongoUri = "<Your Atlas Connection String>";
+
+            // The IMongoClient is the object that defines the connection to our
+            // datastore (Atlas, for example)
+            IMongoClient client;
+
+            // An IMongoCollection defines a connection to a specific MongoDB
+            // collection. Your app may have one or many different IMongoCollection
+            // objects.
+            IMongoCollection<Recipe> collection;
+
+            // Note that you must define the *type* of data stored in the
+            // IMongoCollection. We have created a class called Recipe at
+            // the bottom of this file that serves as a "mapping class" -- the 
+            // driver maps the C# class to the BSON stored in MongoDB.
+
+            // Using mapping classes is strongly advised, but if you
+            // don't create them, you can always use the more generic BsonDocument
+            // type.
+
+            try
+            {
+                client = new MongoClient(mongoUri);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("There was a problem connecting to your " +
+                    "Atlas cluster. Check that the URI includes a valid " +
+                    "username and password, and that your IP address has " +
+                    $"been whitelisted. Message: {e.Message}");
```

Comment:
I think we're changing the "whitelisted" term. Maybe you can check in with someone on CET to find out what term they'll be using.

Reply:
As per https://jira.mongodb.org/browse/INIT-85, it looks like "whitelist" ==> "Access List"

Reply:
Good to see.  My team has been holding out on this till we get a clear signal from the Co on what language we'll be using.  I have a feeling the Docs team will play a role in that decision/cutover.